lib/active_model/serializer/adapter/json.rb

```diff
@@ -1,47 +1,128 @@
 class ActiveModel::Serializer::Adapter::Json < ActiveModel::Serializer::Adapter
-        extend ActiveSupport::Autoload
```

Comment:
It's like that in master on purpose.. to make the diffs easier to read.. previous comment